modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.jsx

```diff
@@ -170,54 +189,68 @@ const AppContextProvider = ({children}) => {
 		};
 
 		const fetchData = async () => {
-			const projectExternalReferenceCode = getAccountKey();
+			try {
+				const projectExternalReferenceCode = getAccountKey();
 
-			if (!projectExternalReferenceCode) {
-				Liferay.Util.navigate(pageRoutes.home());
-			}
-
-			const user = await getUser(projectExternalReferenceCode);
+				if (!projectExternalReferenceCode) {
+					Liferay.Util.navigate(pageRoutes.home());
+				}
 
-			if (user) {
-				const isValid = await isValidPage(
-					client,
-					user,
-					projectExternalReferenceCode,
-					ROUTE_TYPES.project
-				);
+				const user = await getUser(projectExternalReferenceCode);
 
-				if (isValid) {
-					let accountBrief = user.accountBriefs?.find(
-						(accountBrief) =>
-							accountBrief.externalReferenceCode ===
-							projectExternalReferenceCode
+				if (user) {
+					const isValid = await isValidPage(
+						client,
+						user,
+						projectExternalReferenceCode,
+						ROUTE_TYPES.project
 					);
 
-					const apiPermission = 
-						user.isOmniAdmin || user.isPartner || user.isStaff;
-
-					if (!accountBrief && apiPermission) {
-						const {data: dataAccount} = await client.query({
-							query: getAccountByExternalReferenceCode,
-							variables: {
-								externalReferenceCode: projectExternalReferenceCode,
+					if (isValid) {
+						let accountBrief = user.accountBriefs?.find(
+							(accountBrief) =>
+								accountBrief.externalReferenceCode ===
+								projectExternalReferenceCode
+						);
+
+						const apiPermission =
+							user.accountInsideOrganization ||
+							user.isOmniAdmin ||
+							user.isPartner ||
+							user.isStaff;
+						if (!accountBrief && apiPermission) {
```

Comment:
put a new line above this if statement so its easier to read

Reply:
I'll adjust, thanks.